src/test/regress/expected/citus_local_tables_mx.out

```diff
@@ -724,5 +724,72 @@ $$);
  (localhost,57638,t,0)
 (2 rows)
 
+-- test adding foreign table to metadata with the guc
```

Comment:
Initial observation: `ALTER FOREIGN TABLE` commands does not seem to work, maybe you can work on that while I continue to my review.
In general, lets make sure that all here works with our tables: https://www.postgresql.org/docs/14/sql-alterforeigntable.html

```SQL
-- !!!!does not send to the command workers!!!!!
-- also the query fails
CREATE SCHEMA sc10;
alter foreign table foreign_table set schema sc10;
SELECT count(*) FROM sc10.foreign_table ;
DETAIL:  on server onderkalaci@localhost:5432 connectionId: 8
ERROR:  relation "sc10.foreign_table_102107" does not exist
CONTEXT:  while executing command on localhost:5432

```